### DISCUSSION TITLE
Please let me hide people starring my repository

### DISCUSSION THREAD

**Discussion by flexagoon** [community/community#130286] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I use the feed to discover new repositories starred by people who I follow, but there's also a feed entry for every single person who stars my repository, which clutters the feed. The "I'm not interested in this repository" button doesn't hide those events from my feed.

**Comment by joshistoast** [comment]
Have you tried disabling the "Stars" option under the feed filters?

**Reply by flexagoon** [reply]
As I said, I want to see when my friends star something, but not when someone stars my repository
